src/hooks/__internal__/useLocale/index.spec.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { mount } from "enzyme";
+
+import useLocale from "./index";
+import I18nProvider from "../../../components/i18n-provider";
+
+describe("useLocale custom hook", () => {
+  it("when no I18nProvider exists should throw an error", () => {
+    const TestComponent = () => {
+      useLocale();
```

Comment:
I thought this would default to the `Context.Provider` default value. If not, we should make the hook also default to `en-gb` so users don't have to wrap their application in `<I18nProvider>` if they want `en-gb`.

Reply:
My bad, this test didn't work. `en-GB` is default. Users don't have to wrap in `<I18nProvider>` if they want `en-GB`. I've added the correct unit test. Thanks